qadence/engines/torch/differentiable_expectation.py

```diff
@@ -232,160 +229,3 @@ def construct_rules(
                 # Since they are constants their gradients are 0.
                 param_to_psr[param_id] = lambda x: torch.tensor([0.0], requires_grad=False)
         return param_to_psr
-
-
-class DifferentiableBackend(Module):
-    """A class to abstract the operations done by the autodiff engine.
-
-    Arguments:
-        backend: An instance of the QuantumBackend type perform execution.
```

Comment:
This one is now one level up right, so it was dead code?

Reply:
we have a "abstract" DifferentiableBackend which is used by both jax and torch. so i guess some remnants werent properly cleaned up

Reply:
resolving so i can merge